scalding-hraven/src/main/scala/com/twitter/scalding/hraven/reducer_estimation/HRavenHistoryService.scala

```diff
@@ -179,7 +181,8 @@ object HRavenHistoryService extends HistoryService {
       } yield toFlowStepHistory(keys, step, tasks)
     }
 
-  private def toFlowStepHistory(keys: FlowStepKeys, step: JobDetails, tasks: Seq[Task]) =
+  private def toFlowStepHistory(keys: FlowStepKeys, step: JobDetails, tasks: Seq[Task]) = {
+    val mapOutputBytes = step.getCounters.getCounter(TaskCounterKey, MapOutputBytesKey).getValue
```

Comment:
wondering if we should be a bit defensive here in case HRaven returns an empty counters list / the map output bytes counter isn't found. We can't really proceed with the hraven reducer estimator without this value but we could throw a better exception than an NPE.